src/features/ssh/components/SshConnectionDialog.tsx

```
@@ -0,0 +1,209 @@
+                  onClick={() => setAuthMethod(m)}
+                  className={`rounded-md px-3 py-1 text-xs font-medium transition-colors ${
+                    authMethod === m
+                      ? "bg-k-accent/20 text-k-accent"
+                      : "text-k-text-tertiary hover:text-k-text-secondary bg-white/5"
+                  }`}
+                >
+                  {m === "key"
+                    ? t("sshAuthKey")
+                    : m === "password"
+                      ? t("sshAuthPassword")
+                      : t("sshAuthAgent")}
```

ESLint

[error] Do not nest ternary expressions.

(no-nested-ternary)